README.md

```diff
@@ -261,3 +267,7 @@ Ref: Heer, Jeffrey (2019). Visualization is not Enough.
 ### [Visualizing for the Non‐Visual: Enabling the Visually Impaired to Use Visualization [PDF]](http://users.umiacs.umd.edu/~elm/projects/vis4nonvisual/vis4nonvisual.pdf)
 
 Ref: Choi, J., Jung, S., Park, D. G., Choo, J., & Elmqvist, N. (2019, June). Visualizing for the Non‐Visual: Enabling the Visually Impaired to Use Visualization. In *Computer Graphics Forum* (Vol. 38, No. 3, pp. 249-260).
+
+### [AutoVizuA11y: A Tool to Automate Screen Reader Accessibility in Charts [PDF]](https://onlinelibrary.wiley.com/doi/10.1111/cgf.15099)
```

Comment:
This isn't technically a link to the PDF, perhaps either use the epdf link from Eurographics (https://onlinelibrary.wiley.com/doi/epdf/10.1111/cgf.15099) or just remove the `[PDF]` tag here.

Reply:
Otherwise, this is looking great! Ready to merge after this and thanks again for the PR.

Reply:
Just changed the link as suggested :)